storage/innobase/os/os0file.cc

```diff
@@ -6203,9 +6203,15 @@ os_file_read_page(
 			}
 		}
 
+		char fdname[FN_REFLEN];
+		snprintf(fdname, sizeof fdname, "/proc/%d/fd/%d", getpid(), file);
+		char filename[FN_REFLEN];
+		my_readlink(filename, fdname, MYF(0));
 		ib::error() << "Tried to read " << n
 			<< " bytes at offset " << offset
-			<< ", but was only able to read " << n_bytes;
+			<< ", but was only able to read " << n_bytes
```

Comment:
The MTR testsuite needs to be checked for any testcases expecting and depending on this output format (try grepping for "Tried to read " in mysql-test/*).

Reply:
```
[root@marcelo-altmann-BUILD-standalone-1 percona-server]# grep -Ri 'Tried to read' mysql-test/ | wc -l
0
```